node/config/def.go

```diff
@@ -107,6 +107,7 @@ func DefaultBoost() *Boost {
 			HttpTransferMaxConcurrentDownloads: 20,
 			HttpTransferStallTimeout:           Duration(5 * time.Minute),
 			HttpTransferStallCheckPeriod:       Duration(30 * time.Second),
+			DealLogDuration:                    0,
```

Comment:
By default lets turn on log cleanup and set it to a month:
```suggestion
			DealLogDurationDays:                    30,
```